docs/architecture/overview.md

```diff
@@ -1,34 +1,34 @@
 # Architecture Overview
 
-The Carbon Aware SDK was built in 3 layers, with each layer having it's own set of responsibilities. 
-1. User-interface layer
+The Carbon Aware SDK is built with three layers where each layer has its own set of responsibilities.
```

Comment:
While we are updating the name of the top level of our architecture anyway, we should consider using the more common term, `tier` over `layer`.  This codebase is a classic `N-tier architecture` and will be more easily identified as such if we use that common language.